src/blog/2019-03-14/announcing-dotnet-framework-support.pug

```diff
@@ -0,0 +1,67 @@
+extends ../../layout/blog-layout.pug
+block content
+    :markdown-it
+    
+        Stryker just stepped up to become the standard for mutation testing .NET. With version 0.10 you can finaly test your tests on .NET Framework projects. This was a highly needed feature since most .NET projects still are running on .NET Framework and not on the new .NET Core.
```

Comment:
Don't use words like "finaly" or "the standard". Let people make those kinds of judgments for themselves. Suggestion:

> Stryker.NET only supported .NET core projects before. With version 0.10, it's now also possible to mutation test .NET (full) framework projects. This was a highly requested feature, since there are still a lot of .NET framework projects out there and we cannot expect everyone to make the switch to .NET Core right away.